File: m/rls/rlb/stdel.c
/* file -  stdel - delete leading substring */
#include "m:\rid\rider.h"
#include "m:\rid\stdef.h"
/* code -  st_del - delete leading substring */
char *st_del(
register char *str ,
size_t cnt )
{ st_mov (str+cnt, str);
  return ( str);
} 

File: m/rls/stdel.r
file	stdel - delete leading substring
include	rid:rider
include	rid:stdef

code	st_del - delete leading substring

  func	st_del
	str : * Char			; target
	cnt : size			; number to delete
	()  : * char			; at string 
  is	st_mov (str+cnt, str)		; squash them out
	reply str			;
  end
